seqr/views/apis/data_manager_api.py

```diff
@@ -578,6 +592,19 @@ def _get_valid_project_samples(project_samples, sample_type):
     return individual_ids
 
 
+def _get_loaded_samples(sample_keys, user):
+    sample_ids = [sample_id for _, sample_id in sample_keys]
+    samples_by_id = AirtableSession(user).get_samples_for_sample_ids(sample_ids, ['PDOStatus', 'SeqrProject'])
+    return [(project, sample_id) for project, sample_id in sample_keys if any(
+        _is_loaded_airtable_sample(s, project) for s in samples_by_id.get(sample_id, [])
+    )]
+
+
+def _is_loaded_airtable_sample(sample, project_guid):
+    return f'https://seqr.broadinstitute.org/project/{project_guid}/project_page' in sample['SeqrProject'] and any(
```

Comment:
Airtable integration is not at all supported for local installations so we do not generally try to make it usable, but I can use an env var here since  we have one

Reply:
👍 It's probably more explicit to make the airtable integration be a property of the environment, though out of scope for this pr.

Reply:
I feel like when we do the integration to programmatically trigger loading in local installs, that will be a good time to make the airtable integration configuration explicit